### PR TITLE
Add tests for absolute paths and aliases

### PR DESCRIPTION
**Pull request by mrseanryan**
Add tests for absolute paths and for aliases.

### CODE REVIEW

**Comment by mrseanryan** [comment]
@pzavolinsky  this PR just adds more tests, to cover aliases and also using absolute paths.

File: example/absolute-paths-2-tsconfigs/package.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+  "scripts": {
+    "build": "tsc -p ./tsconfig.json",
+    "test": "npm i && npm run build && npm run test:test",
+    "test:test": "../../bin/ts-unused-exports ./tsconfig.json"
+  },
+  "dependencies": {
+    "typescript": "^3.6.4"
+  },
+  "devDependencies": {
+    "jest": "^24.9.0"
```
Comment (pzavolinsky) [inline example/absolute-paths-2-tsconfigs/package.json]: We should drop this dependency.

Comment (mrseanryan) [reply example/absolute-paths-2-tsconfigs/package.json]: done

**Comment by mrseanryan** [comment]
hi @pzavolinsky  do you mind if I go ahead and merge?

**Comment by pzavolinsky** [comment]
Sure, go ahead

**Pull request merged**